packages/server/lib/modes/run.coffee

```diff
@@ -355,6 +355,7 @@ openProjectCreate = (projectRoot, socketId, options) ->
     onError: (err) ->
       console.log("")
       console.log(err.stack)
+      console.log(chalk.yellow(err.details))
```

Comment:
this is a big change that likely affects a lot of things if I'm understanding it correctly

Reply:
Yeah it's a fairly significant change, but the alternatives seemed pretty hacky to me (details elements in error messages and then stripping them out before logging, trying to figure out which chunks should be collapsed on the desktop-gui side). Let me know if it seems like I should be making a lower impact change here.

Reply:
I'm not sure until I see what the before/after or intention of the changes is. Can you take screenshots to post... or maybe just schedule a pairing / review session with me and/or chris